src/components/ShareCurrentScenarioButton/labels.js

```diff
@@ -13,6 +19,13 @@ export const getShareScenarioDialogLabels = (t, currentScenarioName, isDirty, ha
       'commoncomponents.dialog.share.button.editModeTooltip',
       'Please save or discard current modifications before changing the scenario access permissions'
     );
+  else if (datasetBasesCannotBeShared.length > 0)
+    tooltip = t(
+      'commoncomponents.dialog.share.button.noAccessToDatasetBasesTooltip',
+      'Cannot share this scenario because you cannot share its base dataset "{{datasetBasesId}}". Please ' +
+        'request the "admin" role on this dataset first, or ask the dataset owner to share this scenario for you.',
+      { datasetBasesId: datasetBasesCannotBeShared[0] }
```

Comment:
If we only need one dataset id here, then maybe we can rename the parameter to `restrictedDatasetId`, and provide directly the dataset id instead of an Array?